src/Models/NonhydrostaticModels/pressure_correction.jl

```diff
@@ -112,7 +15,7 @@ function compute_pressure_correction!(model::NonhydrostaticModel, Δt)
     foreach(mask_immersed_field!, model.velocities)
     fill_halo_regions!(model.velocities, model.clock, fields(model))
 
-    #correct_boundary_mass_flux!(model)
+    correct_boundary_mass_flux!(model)
```

Comment:
perhaps we can find a desciptor that is more specific than "correct"? Perhaps `enforce_open_boundary_mass_conservation` or something along those lines (eg indicate that this has to do with _open_ boundaries, and mass fluxes / mass conservation)